CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 include(${CMAKE_SOURCE_DIR}/cmake/Modules/misc.cmake)
 
 # This is done here so that relative search paths are more reasnable
-find_package(Irrlicht)
+find_package(Irrlicht REQUIRED)
```

Comment:
Irrlicht *headers* are required to build the server.
Doesn't this search for Irrlicht libraries too?